src/_sass/home2.scss

```
@@ -295,6 +297,7 @@
 .Home2-features-contentList {
   list-style: none;
   margin-left: 0;
+  overflow: scroll;
```

I didn't see a horizontal scroll when I played around in browser, but just to ensure there's only vertical scroll...

```suggestion
  overflow-y: scroll;
```